.github/workflows/build.yml

```
@@ -204,12 +233,35 @@ jobs:
       - name: Package for Linux DEB
         run: npm run package:linux:deb
 
+      - name: Generate Linux auto-updater metadata
+        run: |
+          # Generate latest-linux.yml for auto-updater
+          DEB_FILE=$(ls dist/*.deb | head -1)
+          DEB_NAME=$(basename "$DEB_FILE")
+          DEB_SIZE=$(stat -c%s "$DEB_FILE")
+          DEB_SHA512=$(sha512sum "$DEB_FILE" | cut -d' ' -f1)
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)